FRAuth/FRAuthTests/FRAuthSwiftTests/FRAuth/DeviceBinding/DeviceAuthenticatorTests.swift

```diff
@@ -337,7 +337,7 @@ class DeviceAuthenticatorTests: FRBaseTestCase {
         let kid = UUID().uuidString
         
         let authenticator = BiometricAndDeviceCredential()
-        authenticator.initialize(userId: userId, prompt: Prompt(title: "", subtitle: "", description: "Description"))
+        let _ = authenticator.initialize(userId: userId, prompt: Prompt(title: "", subtitle: "", description: "Description"))
```

Comment:
I would move all the initialize calls to the original initialization for the object. Something like this:

```let authenticator = BiometricAndDeviceCredential().initialize(userId: userId, prompt: Prompt(title: "", subtitle: "", description: "Description"))```

Reply:
Or add a default initializer on the protocol that accepts those values so that you can init the object with all that in one call.

Reply:
@george-bafaloukas-forgerock changed initialize to return Void.